data/strings/types_strings.txt

```diff
@@ -4310,6 +4213,7 @@
 
   [type.amenity.vending_machine.parking_tickets]
     en = Parking Tickets
+    en-GB = Parking Meter
```

Comment:
Not sure what the semantics are, "parking ticket" is the notice the authorities put on your car to fine you for parking too long or not paying.   If this means the machine you interact with to pay, Parking Meter is ok for en_US.

Reply:
changed to parking meter for all en